.travis.yml

```diff
@@ -61,13 +61,22 @@ jobs:
         - pip install black
       script:
         - black --check ./
+    - stage: test
+      name: "copr build test"
+      script:
+        - export SPEC=python-iml-manager.spec
+        - export SRPM_TASK=iml-srpm
```

Comment:
What does the SRPM_TASK variable indicate?

Reply:
Added in a PR for the Dockerfiles. 

It allows the user to override what make task is used to produce the srpm.